front/lib/api/assistant/configuration.ts

```diff
@@ -1149,6 +1155,42 @@ export async function createAgentActionConfiguration(
         forceUseAtIteration: action.forceUseAtIteration,
       };
     });
+  } else if (action.type === "process_configuration") {
+    return frontSequelize.transaction(async (t) => {
```

Comment:
Definitely outside of the scope of this PR but this could live in it's own class.

Reply:
yes that's the goal